dvc/repo/experiments/queue/celery.py

```diff
@@ -295,6 +295,21 @@ def get_result(
         # out of the active task list, and needs to be loaded here.
         return self._get_done_result(entry, timeout)
 
+    def wait(self, revs: Collection[str]) -> None:
+        """Block until the specified tasks have completed."""
+        import time
+
+        revs = [revs] if isinstance(revs, str) else revs
+        results = self.match_queue_entry_by_name(
+            revs, self.iter_queued(), self.iter_done(), self.iter_failed()
+        )
+        for entry in results.values():
+            if not entry:
+                continue
+            while not self.proc.get(entry.stash_rev):
+                time.sleep(1)
```

Comment:
Is this only used in tests? I'll suggest at least mocking these `sleep()` in tests to `time.sleep(0.001)`.